Source/LibationCli/Options/ExportOptions.cs

```diff
@@ -29,26 +27,38 @@ class Options
 		}
 		*/
 		#endregion
-		[Option(shortName: 'x', longName: "xlsx", SetName = "xlsx", Required = true)]
+		[Option(shortName: 'x', longName: "xlsx", HelpText = "Microsoft Excel Spreadsheet", SetName = "Export Format")]
```

Comment:
This uses `SetName` and SetDownloadStatusOptions uses `Group`. Are they the same? Similar?

Reply:
Simmilar. Both mean that the options are mutually exclusive, but at least one option in a `Group` is required and all options in a `Set` are optional.

I did screw this up though, and the three format options should be in different sets.